chanreg.md

```diff
@@ -12,30 +12,31 @@ with two `#` characters are informal channels.
 ### Project Channels
 
 Project channels are reserved for on-topic channels as per our [on-topic
-projects], represented by owners with an official claim to that name. A
-registered project as per the [registration below] can claim ownership over
-project channels bearing the group's names or name prefixes. For example,
-Libera.Chat itself owns the channel `#libera` and all channels beginning with
-`#libera-`.  As per our [policies](/policies), these channels do not expire.
+projects](#on-topic-projects), represented by owners with an official claim
```

Comment:
I prefer if the opening and closing `[` are on the same line, not because it's not valid markdown, but because highlighting works better with that